packages/fe-container/src/components/top-bar/TopBar.tsx

```diff
@@ -29,6 +29,12 @@ export const TopBar: React.FC = () => {
   const configuration = useContext(ConfigurationContext)
   const user = useContext(UserContext)
   const mustShowBurgerIcon = (configuration?.plugins || []).length > 1
+  const mustShowHelpIcon = configuration?.helpMenu && true
```

Comment:
!== undefined instead of && true

you can consider to move this logic inside the helpicon component